packages/ui/.eslintrc.js

```diff
@@ -1,27 +1,37 @@
 module.exports = {
-  root: true,
-  env: {
-    node: true,
-  },
+  env: { node: true },
+  plugins: ['@typescript-eslint', 'import'],
   extends: ['eslint:recommended'],
-  ignorePatterns: ['dist'],
+  ignorePatterns: [
+    '.eslintrc.js',
+    'coverage',
+    'dist',
+    'node_modules',
+    '__tests__',
+  ],
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 2020,
+    ecmaVersion: 12,
     sourceType: 'module',
-    allowImportExportEverywhere: true,
+    project: ['tsconfig.json'],
+    tsconfigRootDir: __dirname,
   },
-  plugins: ['import'],
   rules: {
-    'import/no-extraneous-dependencies': [
+    'no-console': 'error',
+    'no-debugger': 'error',
+    // prefer `typescript-eslint` rule
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': [
       'error',
       {
-        packageDir: ['.', '../..'],
+        argsIgnorePattern: '_',
+        varsIgnorePattern: '_',
+        ignoreRestSiblings: true,
       },
     ],
```

Comment:
Added some small linting rules to verify linting was working in the _ui_ package